src/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Tilt
 layout: home
+has_calendly: true
```

Comment:
FYI — Instead of having the Calendly JS inline, this bool in the header will inject it into the head of the HTML.